kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -226,6 +226,11 @@ func (options *SubmitJobOptions) Validate() error {
 		if submissionMode != rayv1.InteractiveMode {
 			return fmt.Errorf("Submission mode of the Ray Job must be set to 'InteractiveMode'")
 		}
+		// BackoffLimit cannot co-exist with InteractiveMode
+		// https://github.com/ray-project/kuberay/issues/3525
```

Comment:
Maybe instead of an issue link which will be closed when this PR is merged and requires a lot more reading to understand, it's better to summarize succinctly why the two aren't allowed together?